Cargo.toml

```diff
@@ -22,9 +22,9 @@ byteorder = "1.4.3"
 log = "0.4.17"
 openssl = "0.10.45"
 protobuf = "=3.2.0"
-serde = "1.0.152"
-serde_derive = "1.0.152"
-serde_json = "1.0.91"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.93"
+mdns-sd = "0.6.1"
```

Comment:
__note:__ Can you please move `mdns-sd` to `dev-dependencies`? It's only used in the example, the lib itself doesn't use\require it.

Reply:
Done 👍🏻

Reply:
Thanks! Just run CI before approving, and it looks like it's failing because of formatting errors: https://github.com/azasypkin/rust-cast/actions/runs/4331908609/jobs/7564051944. Would you mind running `cargo +nightly fmt --all`, `cargo clippy --all-targets -- -D warnings` and `cargo test` and handling the remaining warnings\errors?

Reply:
Thank you, for the explanation!
Committed the changes.
Maybe #12 can also be closed?